cmd/cmd.go

```diff
@@ -78,38 +78,61 @@ func CreateHandler(cmd *cobra.Command, args []string) error {
 
 	for i := range modelfile.Commands {
 		switch modelfile.Commands[i].Name {
```

Comment:
Does this `switch` only have one `case` now?

Reply:
That was always the case. This is preferable than `if name == "model" || name == "adapter" || ... {`

Alternatively, this can be `if slices.Contains([]string{"model", "adapter", ...}, name)`